circle.yml

```diff
@@ -11,3 +11,9 @@ dependencies:
 test:
   override:
     - bash test.sh
+  post:
```

Comment:
Hey @gucong3000 Can you explain to me what this post hook does? I'm horrbile with bash.

Reply:
copy ./test/**/*  to here:
![image](https://user-images.githubusercontent.com/1171691/33654360-ce4520ee-daaa-11e7-92f5-fc225c472b30.png)